build/repo.targets

```diff
@@ -5,6 +5,7 @@
     <HtmlJobs>-j "$(RepositoryRoot)src/Benchmarks/benchmarks.html.json"</HtmlJobs>
     <JsonJobs>-j "$(RepositoryRoot)src/Benchmarks/benchmarks.json.json"</JsonJobs>
     <MultiQueryJobs>-j "$(RepositoryRoot)src/Benchmarks/benchmarks.multiquery.json"</MultiQueryJobs>
+    <SignalRJobs>-j $(RepositoryRoot)src/Benchmarks/signalr.json</SignalRJobs>
```

Comment:
Where do you define the table to store the results?

Reply:
Odd, I must have accidentally deleted it